packages/InteractiveProfilingTool-Tests.package/IPTReportTestCase.class/instance/buildDummyClass.st

```diff
@@ -0,0 +1,10 @@
+running
+buildDummyClass
```

Comment:
Verstehe ich das richtig, dass diese Methode während der Testausführung in der Umgebung eine neue Klasse erzeugt? Ist das nicht eine potenzielle Problemquelle (und birgt die Gefahr, dass in meinem lokalen Image der Zustand ungewollt verändert wird)?

Reply:
Ja das ist richtig. Wir haben leider noch keinen besseren Weg gefunden dies zu testen. Zur Zeit sollte es jedoch nicht so problematisch sein. Die Klasse wird ja explizite nur für die Tests verwendet. Sollte ein Test fehlschlagen stellen wir in der `setUp` Methode sicher, dass die Klasse die wir zum Test benötigen im Image liegt.

Reply:
Wäre aber vielleicht angebracht, in `tearDown` die Dummy-Klasse auch wieder zu löschen…

Reply:
@Alexander-Dubrawski (sagt @tillprochaska) - wie siehts aus?